build/android/bytecode/java/org/brave/bytecode/BraveClassAdapter.java

```diff
@@ -38,6 +38,8 @@ public static ClassVisitor createAdapter(ClassVisitor chain) {
         chain = new BraveToolbarLayoutClassAdapter(chain);
         chain = new BraveToolbarManagerClassAdapter(chain);
         chain = new BraveTopToolbarCoordinatorClassAdapter(chain);
+        chain = new BraveSiteSettingsDelegateAdapter(chain);
```

Comment:
nit: we started naming it *ClassAdapter, I believe we should keep doing it for consistency.